### ISSUE TITLE
The diphthongs in the 8-bit variant are very difficult to distinguish by ear

### ISSUE THREAD

**Issue by tsbockman** [MaiaVictor/DesuHex#4]
The diphthongs `ai`, `ei`, `aw`, `ow`, and even `ui` all have pronunciations that overlap with those of the core single vowels `a`, `e`, `i`, `o`, `u` in some way. Even when they are distinguished, the difference is often subtle enough that people aren't very good at consistently pronouncing and hearing it.

I suggest replacing `t` in the 8-bit starting consonant table with `h` and replacing the diphthongs `ai`, `ei`, `ui`, `aw`, `ow` in the vowel table with with `at`, `et`, `it`, `ot`, and `ut`.

That just leaves `oi`, which is the easiest of the diphthongs in the table to distinguish from the single vowels.

**Comment by VictorTaelin** [comment]
Why a `t` exactly? Being a Portuguese speaker I'm not sure what the sound of that would be, can I see an example? Perhaps an `s` or `r` would be more effective.

**Comment by tsbockman** [comment]
Using [`s`](https://en.wikipedia.org/wiki/File:Voiceless_alveolar_sibilant.ogg) to end some syllables and [`h`](https://en.wikipedia.org/wiki/File:Voiceless_glottal_fricative.ogg) to start others creates a written ambiguity because you already used [`sh`](https://en.wikipedia.org/wiki/File:Voiceless_palato-alveolar_sibilant.ogg) as a separate symbol: `pasha` could be `pa`-`sha` or `pas`-`ha`. There would be no way to tell from how it's spelled which bit pattern is intended.

Using `r` would work in written form. However, when `h` appears directly after another consonant, it often becomes silent, or just slightly alters the next vowel. For example, there is no [`h` sound](https://en.wikipedia.org/wiki/File:Voiceless_glottal_fricative.ogg) in [`rhodes`](https://forvo.com/word/rhodes/); it sounds like `rodes`. This same issue occurs in combination with many other consonants in English.

However, `t` specifically does not cause this problem: `patha` would be pronounced as either `path`-`a` or `pat`-`ha`, both of which can be easily distinguished from all other sounds currently in Bitspeak. (Just like `sh` and `ch`, `th` in English normally combine with a unique sound: either a [voiceless](https://en.wikipedia.org/wiki/Voiceless_dental_fricative) or [voiced](https://en.wikipedia.org/wiki/Voiced_dental_fricative) dental fricative.)

Your other ending consonant, `n`, is also OK: I think that `panha` would always be pronounced `pan`-`ha`.

**Comment by tsbockman** [comment]
I was thinking about the problem of a silent `h` in `rh` a bit more, and realized that it mostly occurs at the beginning of words. Since Bitspeak would only ever generate `rh` in the middle of words, I think people would pronounce the `h`.

So, while I still think `t` is better, `r` is fine. But, definitely don't use `s`.

**Comment by VictorTaelin** [comment]
TBH in general I think the 8-bit version is just worse and full of ambiguities. I'm not sure it can be saved. The 6-bit one is the most accurate I can think of, and even it still has some minor issues.

**Comment by tsbockman** [comment]
You're right that the 6-bit one is better. And, I thought of a simple way to extend it so that it can do everything the 8-bit version does, without the problems:

Add `h` (or `w`) as a consonant that represents nothing, and `o` as a vowel that represents nothing. Examples:

Hexadecimal `3b` is `0011 1011` in binary, which becomes `dihe` in Bitspeak.
Hexadecimal `9127` is `1001 0001 0010 0111` in binary, which becomes `vakijo` in Bitspeak.

In this way, any even number of bits can be expressed precisely in Bitspeak, while retaining all the advantages of your current 6-bit scheme. The existing 8-bit scheme can simply be removed, as it is unnecessary.

**Comment by johnchandlerburnham** [comment]
I used a terminal `n` in Syllabytes for 8-bit https://github.com/johnchandlerburnham/syllabytes, but this requires a `'` character to disambiguate from an initial `n`
